cspell.json

```diff
@@ -85,6 +85,7 @@
     "combobox",
     "combobutton",
     "createmodal",
+    "createsidepanel",
```

Comment:
```suggestion
```
We don't need to add any files to the cspell file, as #5202 takes care of the avt files.